This dataset has two files containing the titles (titles.csv) and the cast (credits.csv) for the title.
Source: https://www.kaggle.com/victorsoeiro/datasets
License: CC0: Public Domain

# Question 1: What does the dataset look like?

In [11]:
import pandas as pd
netflix_credits = pd.read_csv("credits.csv")
netflix_credits.index = netflix_credits.index +1
print(netflix_credits.head(),"\n",netflix_credits.tail(),"\ncredits_table", netflix_credits.shape)

   person_id       id             name                character   role
1       3748  tm84618   Robert De Niro            Travis Bickle  ACTOR
2      14658  tm84618     Jodie Foster            Iris Steensma  ACTOR
3       7064  tm84618    Albert Brooks                      Tom  ACTOR
4       3739  tm84618    Harvey Keitel  Matthew 'Sport' Higgins  ACTOR
5      48933  tm84618  Cybill Shepherd                    Betsy  ACTOR 
        person_id         id                name     character      role
77797     736339  tm1059008    Adelaida Buscato     María Paz     ACTOR
77798     399499  tm1059008  Luz Stella Luengas  Karen Bayona     ACTOR
77799     373198  tm1059008         Inés Prieto         Fanny     ACTOR
77800     378132  tm1059008        Isabel Gaona        Cacica     ACTOR
77801    1950416  tm1059008      Julian Gaviria           NaN  DIRECTOR 
credits_table (77801, 5)


77801 credits of actors and directors on Netflix titles with 5 columns contains:
1. person_id: The actor or director's id on JustWatch (unique identifier)
2. id: The title id on JustWatch
3. name: The actor or director's name
4. character: The charactor name
5. role: ACTOR or DIRECTOR

In [13]:
netflix_titles = pd.read_csv("titles.csv")
netflix_titles.index = netflix_titles.index +1
print(netflix_titles.head(),"\ntitles_table", netflix_titles.shape)

         id                                title   type  \
1  ts300399  Five Came Back: The Reference Films   SHOW   
2   tm84618                          Taxi Driver  MOVIE   
3  tm154986                          Deliverance  MOVIE   
4  tm127384      Monty Python and the Holy Grail  MOVIE   
5  tm120801                      The Dirty Dozen  MOVIE   

                                         description  release_year  \
1  This collection includes 12 World War II-era p...          1945   
2  A mentally unstable Vietnam War veteran works ...          1976   
3  Intent on seeing the Cahulawassee River before...          1972   
4  King Arthur, accompanied by his squire, recrui...          1975   
5  12 American military prisoners in World War II...          1967   

  age_certification  runtime                                       genres  \
1             TV-MA       51                            ['documentation']   
2                 R      114                           ['drama', 'crim

5850 shows/movies titles on Netflix with 15 columns contains: 

1. id: The title ID on JustWatch.
2. title: The name of the title.
3. show_type: TV show or movie.
4. description: A brief description.
5. release_year: The release year.
6. age_certification: The age certification.
7. runtime: The length of the episode (SHOW) or movie.
8. genres: A list of genres.
9. production_countries: A list of countries that produced the title.
10. seasons: Number of seasons if it's a SHOW.
11. imdb_id: The title ID on IMDB.
12. idb_score: Score on IMDB.
13. imdb_votes: Votes on IMDB.
14. tmdb_popularity: Popularity on TMDB.
15. tmdb_score: Score on TMDB. 